Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.Profiling;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    public enum TextureProperties
+    {
+        Name,
+        Format,
+        Shape,
+        ReadWrite,
+        StreamingMipMaps,
+        MinMipMapLevel,
+        Resolution,
+        SizeOnDisk,
+        Path,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;                 // Manually Choose Folders to do the FindAssets search within.
+
+
+        static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                // new PropertyDefinition { type = PropertyType.Description, name = "Texture Analysis", longName = "Results of Texture Analysis "},
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Show Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Show Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "Shape", longName = "Show Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ReadWrite), format = PropertyFormat.String, name = "Writeable", longName = "Show Texture ReadWrite" },
```

Comment:
please update description to "Readable"

Reply:
Solved in [Tweaks to Texture Module] commit.